rpgJavaInterpreter-core/src/test/resources/data/ds/MUTE12_16.rpgle

```diff
@@ -86,7 +86,7 @@
     MU* VAL1(£JaxWT5) VAL2(12.345) COMP(EQ)
      C                   SETON                                        50
       *
-    MU* VAL1(£JaxWT6) VAL2(-0.12345678) COMP(EQ)
+    MU* VAL1(£JaxWT6) VAL2(-0.12345678) COMP(NE)
```

Comment:
This is not the solution, take in account that MU* stands for mute annotation, and it is exactly the value of that variable in as400

Reply:
As I explained in the description, the test MUTE12_16 was wrong. The variable £JaxWT6 is equal to 0 and it's compared to the constant "-0.12345678". They can't be equal. Without this fix, the result was wrong. If you look at the Screenshot1 (branch develop), you will see that even if Jariko is comparing the different values (value1 = 0 and value2 = -0.12345678) returned true as result when was testing if the variables were Equal. The problem is that it was comparing an intValue with a DecimalValue. In the Screenshot2 (current branch) you can see that now there are 2 Decimal values and the test return true as result when it's testing if the variables are Different.

![Screenshot1](https://github.com/smeup/jariko/assets/32836304/2ce54250-96ae-49e2-81ff-360f85eef408)

![Screenshot2](https://github.com/smeup/jariko/assets/32836304/9cf64629-3764-4628-bf3c-91e8e7d2752c)

Reply:
Ok, but in this case a NE(argument) test is much less significant than an EQ(argument) test, because NE(argument)  means infinite values except the argument and then: Ok, this test is wrong, but make it better with EQ (like before) instead of NE.

Reply:
done